API_Gateway_Automation/api-gateway-lambda-integration.yaml

```diff
@@ -0,0 +1,121 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: AWS API Gateway with a Lambda Integration
+
+Resources:

```

Comment:
The yaml file will create the resources given the parameters you have, next step would be adding a shell or python script to deploy your yaml file.

Reply:
I have left instructions in the README how to deploy in Git Bash.  Please see above as explain again:

In the README file explains how to implement, deployment, and test that the Yaml file as follows in Git Bash:

AWS API Gateway with a Lambda Integration
###Validation

$ aws cloudformation validate-template
--template-body file://api-gateway-lambda-integration.yaml

###Installation Deploy

$ aws cloudformation deploy
--stack-name lambda-api
--template-file api-gateway-lambda-integration.yaml
--capabilities CAPABILITY_IAM

###Testing the API Gateway Once our API Gateway is deployed, testing simply involves making a request to the endpoint:

$ http -v POST
[https://ld47kkph0k.execute-api.us-west-2.amazonaws.com/v0/lambda](https://ld47kkph0k.execute-api.us-west-2.amazonaws.com/v0/lambda%5C)
POST /v0/lambda HTTP/1.1
Accept: /
Accept-Encoding: gzip, deflate
Connection: keep-alive
Content-Length: 0
Host: ld47kkph0k.execute-api.us-east-1.amazonaws.com
User-Agent: HTTPie/1.0.2